src/util/test.rs

```diff
@@ -56,3 +59,14 @@ pub fn bootstrap_cache_tmp_file() -> OsString {
     path.push(file_name);
     path.into()
 }
+
+/// Constructs event sender/receiver pair.
+pub fn event_sender() -> (CrustEventSender, Receiver<Event>) {
```

Comment:
A method named `sender` returning a `receiver` is interesting :P .. maybe call it `event_channel()` or something else better if you agree.

Reply:
mhm, sounds good to me